openqa-hw-tests/power

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+rte_ip=$1
+pikvm_ip=$2
+action=$3
+
+function pikvm_cmd() {
+    sshpass -proot \
+            ssh -q -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null "root@$pikvm_ip" \
+                "$@"
+}
+
+function rte_cmd() {
+    sshpass -pmeta-rte \
+            ssh -q -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null "root@$rte_ip" \
+                "$@"
+}
+
+function setup_video() {
+    if pikvm_cmd v4l2-ctl --get-dv-timings | grep -q 'Active width: 0'; then
+        echo "Setting EDID"
+        pikvm_cmd v4l2-ctl --set-edid 'file=$HOME/openqa/edid.1024x768'
```

Comment:
This needs to be done once after boot, I have a systemd unit for it: https://github.com/QubesOS/tests-hw-setup/blob/5de30dfe12d0e9399b1d085fda738c1b0fc992b5/states/openqa-worker.sls#L273-L288

Reply:
Thanks, but having maximum resolution on boot may be useful to run some manual tests, I didn't want to fix it to 1024x768.